Sources/BraintreeShopperInsights/BTCreateCustomerSessionApi.swift

```diff
@@ -0,0 +1,106 @@
+import UIKit
+
+#if canImport(BraintreeCore)
+import BraintreeCore
+#endif
+
+/// The API used to create a customer session using the `CreateCustomerSession` GraphQL mutation.
+/// - Warning: This feature is in beta. It's public API may change or be removed in future releases.
+class BTCreateCustomerSessionApi {
+    
+    // MARK: - Properties
+    
+    /// Exposed for testing to get the instance of `BTAPIClient`
+    var apiClient: BTAPIClient
+    
+    /// :nodoc: This typealias is exposed for internal Braintree use only. Do not use. It is not covered by Semantic Versioning and may change or be removed at any time.
+    typealias CreateCustomerSessionResult = (String?, Error?) -> Void
+    
+    // MARK: - Init
+    
+    /// Creates a `BTCreateCustomerSessionApi`
+    /// - Parameters:
+    ///    - apiClient: A `BTAPIClient` instance
+    /// - Warning: This init is beta. It's public API may change or be removed in future releases. This feature only works with a client token.
```

Comment:
Since this class should be internal to the SDK, do we need the beta warning for each of these comments?

Reply:
ahhh yep that's true i don't think we need it, addressed in 364600e2cd5aa4829306bb6c114a58e2ea6b5de1